src/Mount.cpp

```diff
@@ -748,31 +747,33 @@ bool Mount::connectToDriver( TMC2209Stepper* driver, const char *driverKind ) {
     _driverFocus->beginSerial(19200);
     _driverFocus->mstep_reg_select(true);
     _driverFocus->pdn_disable(true);    
-    bool _UART_Rx_connected = false;
     #if UART_CONNECTION_TEST_TXRX == 1
-        _UART_Rx_connected = connectToDriver( _driverFocus, "Focus" );
-        if (!_UART_Rx_connected) {
-            digitalWrite(FOCUS_EN_PIN, HIGH);    //Disable motor for safety reasons if UART connection fails to avoid operating at incorrect rms_current
-        }
+      bool UART_Rx_connected = false;
+      UART_Rx_connected = connectToDriver( _driverFocus, "Focus" );
+      if (!UART_Rx_connected) {
+          digitalWrite(FOCUS_EN_PIN, HIGH);    //Disable motor for safety reasons if UART connection fails to avoid operating at incorrect rms_current
+      }
     #endif
     _driverFocus->toff(0);
     #if USE_VREF == 0  //By default, Vref is ignored when using UART to specify rms current.
         _driverFocus->I_scale_analog(0);
     #endif
     LOGV2(DEBUG_STEPPERS, F("Mount: Requested Focus motor rms_current: %d mA"), rmscurrent);
-    _driverFocus->rms_current(rmscurrent, 0.01f); //holdMultiplier = 1 to set ihold = irun
+    _driverFocus->rms_current(rmscurrent, 0.1f); //holdMultiplier = 1 to set ihold = irun
     _driverFocus->toff(1);
     _driverFocus->en_spreadCycle(FOCUS_UART_STEALTH_MODE == 0);
     _driverFocus->blank_time(24);
     _driverFocus->microsteps(FOCUS_MICROSTEPPING == 1 ? 0 : FOCUS_MICROSTEPPING);   // If 1 then disable microstepping
     _driverFocus->TCOOLTHRS(0xFFFFF);  //xFFFFF);
```

Comment:
`//xFFFFF);` ?

Reply:
No idea. The hardware people wrote that :-)